examples/get_current_viewer.py

```diff
@@ -20,3 +21,14 @@
 
 # get that reference again
 viewer = napari.current_viewer()
+
+# work with the viewer
+x = np.arange(256)
+y = np.arange(256).reshape((256, 1))
+# from: https://botsin.space/@bitartbot/113553754823363986
+image = (-(~((y - x) ^ (y + x)))) % 11
```

Comment:
Maybe we could start promoting writing functions by putting such parts in functions with proper docstring? 

It will increase example length, but increase readability. And possibly in the future we will get scientific code with properly named functions instead of a wall of text?